tools/selection_tools.cpp

```diff
@@ -255,6 +255,65 @@ void CloseSelection(Mesh* obj)
 	CloseSelection (obj->selector());
 }
 
+void RestrictSelectionToSubset(Mesh* obj, int si)
+{
+	SubsetHandler& sh = obj->subset_handler();
+	Selector& sel = obj->selector();
+
+//	Store selected elements that are contained in the specified subset
+	std::vector<Vertex*> vVertices;
```

Comment:
While of course a matter of taste, The 'v' prefix is unnecessary, I'd say, since the type is clear. There are no prefixes on many other variables, so this is probably slightly inconsistent.